docs/DeveloperGuide.md

```diff
@@ -1331,22 +1344,142 @@ testers are expected to do more *exploratory* testing.
+      size of student list)<br>
+      Expected: No student is deleted. Error details shown in the status message.
+
+1. Deleting a lesson
+
+   1. Prerequisites: List all classes using the `list-class` command. The first class contains at least one lesson.
+
+   1. Test case: `delete-lesson c\1 l\1`<br>
+      Expected: First lesson in first class has been deleted from the list. Details of the deleted lesson shown in the
+      status message.
+
+   1. Test case: `delete-lesson c\1 l\0`<br>
+      Expected: No lesson is deleted. Error details shown in the status message.
+
+   1. Other incorrect delete commands to try: `delete-lesson`, `delete-lesson x` (where x is larger than the
+      number of lesson in a specific class)<br>
+      Expected: No lesson is deleted. Error details shown in the status message.
+
+### Displaying a student's statistics
+
+1. Displaying a student's statistics
+
+   1. Prerequisites: List all classes and students using the `list` command. Using default tutor's pet data.
+
+   1. Test case: `stats c\1 s\1`<br>
+      Expected: Statistics for the first student in the first class of the student list and class list respectively
+      found. Details of the statistics shown in the status message.
+
+   1. Test case: `stats c\1 s\4`<br>
+      Expected: Statistics not found. Reason stated in the status message.
+
+   1. Other incorrect display stats commands to try: `stats c\1 s\a`, `stats c\1 s\x`, `...` (where x is larger than the
+      size of student list)<br>
+      Expected: Statistics not found. Error details shown in the status message.
+
+### Displaying a lesson's venue
+
+1. Displaying a lesson's venue
+
+   1. Prerequisites: List all classes using the `list-class` command. Using default tutor's pet data.
+
+   1. Test case: `display-venue c\1 l\1`<br>
+      Expected: Venue for the first lesson in the first class of the class list found. Details of the venue shown in
+      the status message.
+
+   1. Test case: `display-venue c\1 l\2`<br>
+      Expected: Venue not found. Error details shown in the status message.
+
+   1. Other incorrect display venue commands to try: `display-venue c\1 l\a`<br>
+      Expected: Venue not found. Error details shown in the status message.
+
+### Editing a class
+
+1. Editing a class
+
+   1. Prerequisites: List all classes using the `list-class` command. At least one class in the class list.
+
+   1. Test case: `edit-class 1 n\Edited Class`<br>
+      Expected: Name of the first class in the class list have been edited to `Edited Class`. Details of edited class
+      shown in the status message.
+
+   1. Test case: `edit-class`<br>
+      Expected: No class is edited. Error details shown in the status message.
+
+   1. Other incorrect edit commands to try: `edit-class 1 n\!`, `edit-class x`
+      (where x is larger than the size of class list)<br>
+      Expected: No class is edited. Error details shown in the status message.
+
+### Finding a class or attendance
```

Comment:
Should this be separated into two sections? 

Otherwise its a little weird why finding student is not planned to be part of the list...
If the intention is to have find student as part of this section as well, perhaps you can put it in the header and have a placeholder somewhere later in the body.

Reply:
Hmm.. maybe we just seperate into two sections since we plan to follow the ordering of UG